database-next/read-and-write.js

```diff
@@ -153,6 +153,24 @@ function toggleStar_wrapped() {
   // [END rtdb_social_star_transaction]
 }
 
+/**
+ * @param {string} uid 
+ * @param {string} key 
+ */
+// [START rtdb_social_star_increment]
+function addStar(uid, key) {
+  const { getDatabase, ServerValue } = require("firebase/database");
```

Comment:
I think the import is just a free function called `increment` and there's no `ServerValue` at all in vNext

Reply:
Yup, thanks for that.